src/components/form/form.tsx

```diff
@@ -96,6 +103,8 @@ export class Form {
 
     public constructor() {
         this.handleChange = this.handleChange.bind(this);
+        this.getCustomErrorMessages = this.getCustomErrorMessages.bind(this);
```

Comment:
We can also get rid of this bind, by using an arrow function instead. We try to use arrow functions instead of binds when possible nowadays, but there's a bunch of old code still using bind, because we used that a lot a few years ago.

Reply:
Although, you don't really have to change that now, because once we've merged this, I'll change the already existing functions in this file anyway, in a separate PR, so I can do this one then too.

Reply:
Ahh yes, you're right! I will commit some changes shortly @adrianschmidt

Reply:
Sounds good! @adrianschmidt